src/Commands/ResendMailCommand.php

```diff
@@ -3,19 +3,31 @@
 namespace Vormkracht10\Mails\Commands;
 
 use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\PromptsForMissingInput;
+use Vormkracht10\Mails\Jobs\ResendMailJob;
+use Vormkracht10\Mails\Models\Mail;
 
-class ResendMailCommand extends Command
+class ResendMailCommand extends Command implements PromptsForMissingInput
 {
-    public $signature = 'mail:resend {uuid} {to} {cc} {bcc}';
+    public $signature = 'mail:resend {uuid} {to?} {cc?} {bcc?}';
 
     public $description = 'Resend mail';
 
     public function handle(): int
     {
-        // TODO: resend mail
+        $mail = Mail::where('uuid', $this->argument('uuid'))->first();
+
+        ResendMailJob::dispatch($mail,
+            ...collect($this->argument())->only(['to', 'cc', 'bcc'])->map(fn ($n) => $n ?? []),
```

Comment:
Hiervoor zorgen dat 'to', 'cc' en 'bcc' bij niet invullen leeg worden gemaakt in de mail die ge-re-send wordt. @dulkoss

Reply:
De map is misschien overbodig vanwege checkFields in de job.

Reply:
Als we hier geen To hebben, dan een $this->components->error geven?